src/EmployerPR/SFA.DAS.EmployerPR/Domain/RoatpProvider/GetRoatpProvidersResponse.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace SFA.DAS.EmployerPR.Domain.RoatpProvider;
+public class GetRoatpProvidersResponse
```

Comment:
What is this class for?

Reply:
The incoming json names the collection as RegisteredProviders, so this accommodates that naming conversion